### PR TITLE
refactor: move js to bottom and move css to top.

### PR DESCRIPTION
**Pull request by appleboy**
* move CSS link to `<head>` section.
* move js link to bottom of `<body>` section.

### CODE REVIEW

File: templates/base/footer.tmpl [hunk 1]
```diff
@@ -27,26 +27,43 @@
...
+	</script>
+{{end}}
+{{if .RequireGitGraph}}
+<!-- graph -->
+	<script src="{{AppSubUrl}}/js/libs/gitgraph.js"></script>
+	<script src="{{AppSubUrl}}/js/draw.js"></script>
+{{end}}
+
+<!-- Stylesheet -->
+<link rel="stylesheet" href="{{AppSubUrl}}/css/index.css?v={{MD5 AppVer}}">
```
Comment (tboerger) [inline templates/base/footer.tmpl]: This should be in the head

File: templates/base/footer.tmpl [hunk 2]
```diff
@@ -27,26 +27,43 @@
 			</div>
 		</div>
 	</footer>
-</body>
+<script src="{{AppSubUrl}}/js/jquery-1.11.3.min.js"></script>
```
Comment (tboerger) [inline templates/base/footer.tmpl]: This should be indented by one level

File: templates/base/head.tmpl [hunk 3]
```diff
@@ -15,40 +17,33 @@
...
-	<link rel="stylesheet" href="{{AppSubUrl}}/css/index.css?v={{MD5 AppVer}}">
-
-	<!-- JavaScript -->
-	<script src="{{AppSubUrl}}/js/semantic-2.2.1.min.js"></script>
-	<script src="{{AppSubUrl}}/js/index.js?v={{MD5 AppVer}}"></script>
 
-	<title>{{if .Title}}{{.Title}} - {{end}}{{AppName}}</title>
-
-	<meta name="theme-color" content="{{ThemeColorMetaTag}}">
+{{if .RequireHighlightJS}}
```
Comment (tboerger) [inline templates/base/head.tmpl]: This should also be indented by one level

File: templates/base/footer.tmpl (hunk 2 above)
Comment (appleboy) [reply templates/base/footer.tmpl]: done

File: templates/base/footer.tmpl (hunk 1 above)
Comment (appleboy) [reply templates/base/footer.tmpl]: done

File: templates/base/head.tmpl (hunk 3 above)
Comment (appleboy) [reply templates/base/head.tmpl]: I refactor head file.

**Comment by lunny** [comment]
build failed.

**Comment by appleboy** [comment]
screen shot 2017-01-18 at 4 33 10 pm

github api error?

**Comment by appleboy** [comment]
@lunny build successfully.

**Comment by lunny** [comment]
LGTM

**Comment by andreynering** [comment]
Trusted LGTM

**Pull request merged**